modules/microshift-kubeconfig-local-access.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_networking/microshift-kubeconfig.adoc
+
+:_content-type: CONCEPT
+[id="local-access_{context}"]
+= Local access kubeconfig file
+
+The local access `kubeconfig` file is generates in the `/var/lib/microshift/resources/kubeadmin/kubeconfig` directory. This configuration file allows access to the local API server using `localhost` and the internal CA of `localhost`. The CA of the local access `kubeconfig` file cannot be used outside of the {product-title} host.
+
+When the cluster within the {product-title} host connects to the API server locally, a certificate is used for validation. The certificate lists `localhost` as the only Subject Alternative Name (SAN). You must use `localhost` to connect to the local API server. The `kubeconfig` file must use the same certificate authority (CA) that signed the certificate.
```

Comment:
This paragraph starts out with "the cluster" as the actor and then gives the user instructions ("you must"). Maybe we should split it up to explain how the cluster does authentication internally, then tell the user to use a specific kubeconfig when trying to access the API server from the same host.